### DISCUSSION TITLE
Anyone using Django Ninja with Django-NextJS package?

### DISCUSSION THREAD

**Discussion by dizzydes** [vitalik/django-ninja#628] [Q&A]
[Link](https://github.com/yourlabs/djnext).

A lot of the setup docs request/use DRF but I think its structure means the response could just be surfaced with Ninja.

**Comment by baseplate-admin** [comment]
Thing is ( [`django-nextjs`](https://github.com/QueraTeam/django-nextjs) / [`djnext`](https://github.com/yourlabs/djnext) ) just proxies the request to the nodejs backend using requests. This can be accomplished using a reverse proxy. ( Using python to proxy request is a horribly slow and a bad idea | Not to mention requests itself is very slow )

You can probably accomplish this yourself by mapping the urls using a reverse proxy service ( `NGiNX` / `Caddy` )
